python-package/lightgbm/sklearn.py

```diff
@@ -43,6 +44,11 @@
         Union[_LGBM_EvalFunctionResultType, List[_LGBM_EvalFunctionResultType]]
     ],
 ]
+_LGBM_ScikitEvalMetricType = Union[
```

Comment:
Please use new defined variable `_LGBM_ScikitEvalMetricType` in the `dask.py` file as well:
https://github.com/microsoft/LightGBM/blob/c8712a9e083bfa0922f6e860f49a8a374816f43e/python-package/lightgbm/dask.py#L408
https://github.com/microsoft/LightGBM/blob/c8712a9e083bfa0922f6e860f49a8a374816f43e/python-package/lightgbm/dask.py#L1040
https://github.com/microsoft/LightGBM/blob/c8712a9e083bfa0922f6e860f49a8a374816f43e/python-package/lightgbm/dask.py#L1166
https://github.com/microsoft/LightGBM/blob/c8712a9e083bfa0922f6e860f49a8a374816f43e/python-package/lightgbm/dask.py#L1337
https://github.com/microsoft/LightGBM/blob/c8712a9e083bfa0922f6e860f49a8a374816f43e/python-package/lightgbm/dask.py#L1492

Reply:
agree, thanks for the suggestion!

just pushed https://github.com/microsoft/LightGBM/pull/5460/commits/347264d76c5c7cc3fc93f01a5fa42a4cbe9d7c15 with that change